services/director/Dockerfile

```diff
@@ -25,7 +25,7 @@ ENV EXTRA_HOSTS_SUFFIX = 'undefined'
 # ----------------------------------------------------------------
 COPY --chown=scu:scu scripts/docker/healthcheck_curl_host.py /healthcheck/healthcheck_curl_host.py
 # will start to run after interval every interval. fails after timeout. fail do not count if during start-period. will do # retries
-HEALTHCHECK --interval=10s --timeout=30s --start-period=1s --retries=3 CMD [ "python3", "/healthcheck/healthcheck_curl_host.py", "http://localhost:8001/v0/" ]
+HEALTHCHECK --interval=10s --timeout=30s --start-period=300s --retries=3 CMD [ "python3", "/healthcheck/healthcheck_curl_host.py", "http://localhost:8001/v0/" ]
```

Comment:
Any special reason for this change in time for the start-period? Should it be applied as well to other services?

Reply:
And also 300s is too much. Wasn't the issue resolved by setting the correct env variables?